src/evaluator.ts

```diff
@@ -181,26 +157,38 @@ export class Evaluator {
     clause: Clause,
     target: Target,
   ): Promise<boolean> {
-    if (!clause.op) {
+    if (!clause?.op || !clause?.values || clause.values.length === 0) {
```

Comment:
You can probably shorten this a bit
```suggestion
    if (!clause?.op || !clause?.values.length) {
```

Reply:
```if (!clause?.op || !clause?.values.length)``` if values is undefined?: ```if (!clause?.op || !clause?.values?.length)```